src/DynamoCore/Graph/Nodes/CustomNodes/Function.cs

```diff
@@ -338,8 +339,20 @@ public Symbol()
         }
 
         /// <summary>
+        ///     Initializes a new instance of the <see cref="Symbol"/> class.
+        /// </summary>
+        [JsonConstructor]
+        public Symbol(IEnumerable<PortModel> inPorts, IEnumerable<PortModel> outPorts) : base(inPorts, outPorts)
+        {
+            ArgumentLacing = LacingStrategy.Disabled;
+            InputSymbol = String.Empty;
```

Comment:
you can change this constructor to pass in the input symbol and then set it here.